src/scss/objects/_buttons.scss

```diff
@@ -10,92 +10,92 @@
+    $btn-outline-textColor                 : $color-content-interactive-tertiary;
+    $btn-outline-border-color              : $color-border-default;
+
+    $btn-ghost-bgColor                     : transparent;
+    $btn-ghost-bgColor--hover              : darken($color-white, $color-hover-01);
+    $btn-ghost-bgColor--active             : darken($color-white, $color-active-01);
+    $btn-ghost-textColor                   : $color-content-interactive-secondary;
+
+    $btn-disabled-bgColor                  : $color-disabled-01;
+    $btn-disabled-textColor                : $color-content-disabled;
+
+    $btn-sizeLarge-padding                 : 14px spacing(x3);
+    $btn-sizeLarge-loading-color           : $color-content-interactive-light;
+
+    $btn-sizeSmall-font-size               : 'body-l';
+    $btn-sizeSmall-padding                 : spacing() spacing(x2);
+
+    $btn-sizeXSmall-font-size              : 'body-s';
+    $btn-sizeXSmall-padding                : 6px spacing();
 
     /**
     * Base button styles – Based on csswizardry.com/beautons
     *
     * 1. Allow us to better style box model properties.
     * 2. Line different sized buttons up a little nicer.
-    * 3. Make buttons inherit font styles.
-    * 4. Force all elements using beautons to appear clickable.
-    * 5. Normalise box model styles.
-    * 6. If the button’s text is 1em, and the button is (3 * font-size) tall, then
-    *    there is 1em of space above and below that text. We therefore apply 1em
-    *    of space to the left and right, as padding, to keep consistent spacing.
-    * 7. Fixes odd inner spacing in IE7.
     */
 
+    // By deafult the button is Primary medium sized button
     .o-btn {
         display: inline-block;                      /* [1] */
         vertical-align: middle;                     /* [2] */
-        font-family: $btn-default-font-family;      /* [3] */
-        @include font-size($btn-default-font-size); /* [3] */
-        cursor: pointer;                            /* [4] */
-        margin: 0;                                  /* [5] */
-        padding: $btn-default-vertPadding $btn-default-hozPadding; /* [5, 6] */
-        overflow: visible;                          /* [7] */
+        @include font-size($btn-default-font-size);
+        cursor: pointer;
+        margin: 0;
+        padding: $btn-default-padding;
         text-align: center;
         font-weight: $btn-default-weight;
-        line-height: 1;
 
-        // You may want to change this
         background-color: $btn-default-bgColor;
         border-radius: $btn-default-borderRadius;
         border: 1px solid transparent;
         user-select: none;
-
-        color: $color-grey-50;
+        color: $btn-default-color;
         text-decoration: none;
 
-        &:hover,
         &:focus {
-            background-color: $btn-default-bgColor--hover;
+            border: 2px solid $btn-default-outline-color;
```

Comment:
I think this should be `outline` rather than `border` as the border affects the position of the button?